README-cfn.md

```diff
@@ -0,0 +1,179 @@
+4) Check that the bucket contains the following files:
+
+```
+s3://<bucket_name> e.g. my-s3-bucket
+          |-- <key_prefix> e.g. development
+          |       |-- lambdas
+          |       |      |-- eks-helper-lambda.zip
+          |       |      +-- alfresco-lambda-empty-s3-bucket.jar
+          |       |      +-- helm-helper-lambda.zip
+          |       |-- scripts
+          |       |      |-- deleteIngress.sh
+          |       |      +-- getElb.sh
+          |       |      +-- helmDeploy.sh
+          |       |      +-- helmIngress.sh
+          |       |      +-- helmInit.sh
+          |       |-- templates
+          |       |      |-- acs.yaml
+          |       |      +-- full-deployment.yaml
+          |       |      +-- full-master-parameters.json
+          |       |      +-- bastion-and-eks-cluster.yaml
+          |       |      +-- efs.yaml
+          |       |      +-- rds.yaml
+          |       |      +-- s3-bucket.yaml
+```
+
+## Deploying DBP EKS with AWS Console
+**Note:** To use the AWS Console, make sure that you've uploaded the required files to S3 as described in [Preparing the S3 bucket for CFN template deployment](#preparing-the-s3-bucket-for-cfn-template-deployment).
+
+* Go to the AWS Console and open CloudFormation
+* Click ```Create Stack```
+* In ```Upload a template to Amazon S3``` choose `/alfresco-dbp-deployment/aws/templates/full-deployment.yaml`
+* Choose a stack name, like `my-dbp-eks`
+* Fill out the parameters. In many cases you can use the default parameters. For some parameter sections
+we will provide some additional information.
+
+**S3 Cross Replication Bucket for storing DBP content store**
+
+```Enable Cross Region Replication for This Bucket``` : Cross Region Replication replicates your data into another bucket. See [Cross-Region Replication](https://docs.aws.amazon.com/AmazonS3/latest/dev/crr.html) for more information.
+
+**DBP Stack Configuration**
+
+```The name of the S3 bucket that holds the templates``` : Take the bucket name from the upload step.
+
+```The Key prefix for the templates in the S3 template bucket``` : Take the `key_prefix` from the upload step.
+
+```The SSL Certificate arn to use with ELB``` : Take the SSL certificate arn for your domains in the hosted zone. For more information about how to create SSL certificates, see the AWS documentation on the [AWS Certificate Manager](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html).
+
+```The domain name``` : Choose the subdomain which will be used for the url e.g. **my-dbp-eks.example.com**. For more information about how to create a hosted zone and its subdomains visit the AWS documentation on [Creating a Subdomain](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html).
+
+```Private Registry Credentials. Base64 encryption of dockerconfig json``` : \
+**Notice:** The credentials are only needed if the DBP images is pulled from quay.io which is currently not the case.
```

Comment:
I think this is the case for some of the images now such as the transformer nodes.